README.md

```diff
@@ -23,3 +30,15 @@ Zoom link: https://zoom-lfx.platform.linuxfoundation.org/meeting/95471930872?pas
 ### Meeting notes
 
 We store ongoing meeting notes in a [Google doc](https://docs.google.com/document/d/1roexHTLCrErYjNT2NEoRsVnn_YNbQzZ1gyXNK8hXR4Q/). 
+
+## The Technical Documentation Assistance Program
+
+The CNCF Tech Doc team has created a program to assist CNCF-affiliated projects in creating effective documentation. [`TechDoc-Assistance-Program.md`](./TechDoc-Assistance-Program.md) contains an outline of this program. 
+
+Project maintainers are asked to understand that:
+- Project documentation resources are *always* underestimated.
+- Documentation is as much a real part of a project as code. 
+- Effective documentation requires a real commitment of resources. 
+- The TechDoc Assistance Program exists to help maintainers allocate these resources effectively, but cannot write documentation for you or 'fix' documentation problems. 
+
+We're passionate about good documentation and we want to help your project succeed. Please work with us by taking this part of your project seriously.
```

Comment:
I would suggest gentler language; see below.

As for the pitch, importance, and realities surrounding documentation, if we do mention anything, we should link out to external resource(s) IMHO.

```suggestion
The TechDocs team can assist CNCF projects analyze and improve their documentation. For details, see TechDocs [Assistance program](./TechDoc-Assistance-Program.md).
```

Reply:
I agree, my wording was a bit provocative.